### DISCUSSION TITLE
Feature Request: Emojis

### DISCUSSION THREAD

**Discussion by megclaypool** [ebullient/obsidian-snippetor#8] [Ideas]
First, your plugin is super cool!

I was wondering if it would be possible to have an input box in which one could paste an emoji to display inside the checkbox in place of the character. For example, it would be cool if I could type - [!] and have it be replaced with a checkbox with ❗ inside.

Thanks again for your hard work making styling more accessible to non-programmers!!! ❤️

**Comment by ebullient** [comment]
That is already possible. ;)

Make sure you use the expand/collapse control on the right to show extended settings for the row. You can use the "Rendered" input box to paste in your emoji, and the following sliders to adjust its size and position.

![image](https://user-images.githubusercontent.com/808713/152530720-10d5606e-393d-44fd-a2a4-6872ad7bbe1f.png)

**Comment by megclaypool** [comment]
Hot *diggity*!!! Thanks! 😀😀😀❤️

**Comment by megclaypool** [comment]
While we're shooting for the moon, is it possible to render an image, either by pasting it in somehow or using a url or something?

**Comment by ebullient** [comment]
This is .. possible.. but a little hacky... 
What you can try:
1. Use a space for the rendered content value
2. In the background field (using text fields instead of color pickers.. that's the palette toggle), paste any valid css background value.

I know that this works, e.g.
`linear-gradient(135deg, transparent 50%, aqua 50%)` to give you a half-filled checkbox. Any other valid background value should work (including a url encoded svg).

![image](https://user-images.githubusercontent.com/808713/152578009-b4c211f0-4197-45f0-987e-61e0446ff163.png)

**Comment by megclaypool** [comment]
Holy css, Batman! Awesome -- thanks!!!
